system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts

```diff
@@ -1,13 +1,14 @@
 ### Cinder Alerts ###
 
 ALERT OpenstackCinderVolumeStuck
-  IF (sum(openstack_stuck_volumes_count_gauge) by (host,status)) > 0
+  IF sum(openstack_stuck_volumes_count_gauge) by (host,status,check) > 0
   FOR 5m
   LABELS {
     service = "cinder",
     severity = "info",
     tier = "openstack",
-    dashboard = "cinder"
+    dashboard = "cinder",
+    meta = "{{ $labels.check }}"
```

Comment:
I suggest `meta = "{{ $value }} volumes"` to show the number of affected stuck volumes